packages/polling-controller/src/PollingController.ts

```diff
@@ -112,24 +122,29 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
      * Executes the poll for a networkClientId
      *
      * @param networkClientId - The networkClientId to execute the poll for
+     * @param options - The options passed to startPollingByNetworkClientId
      */
-    abstract executePoll(networkClientId: NetworkClientId): Promise<void>;
+    abstract executePoll(
```

Comment:
This is out of scope for this PR, but question here: is it useful to execute the polling outside of the controller? Should this be `protected`?

Reply:
had issues with `protected`, see [here](https://github.com/MetaMask/core/pull/1736#discussion_r1340166105)